src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -5324,8 +5324,7 @@ public function seek( $bookmark_name ): bool {
 		 */
 		if ( 'backward' === $direction ) {
 			/*
-			 * Instead of clearing the parser state and starting fresh, calling the stack methods
-			 * maintains the proper flags in the parser.
+			 * When moving backward, stateful stacks should be cleared.
 			 */
 			foreach ( $this->state->stack_of_open_elements->walk_up() as $item ) {
 				if ( 'context-node' === $item->bookmark_name ) {
```

Comment:
doesn't this also need to go? I can’t remember, but I thought that we stopped pushing the context node onto the stack of open elements, and likely forgot to remove this when we did. same deal a few lines below.

Reply:
I've removed a few special cases that handled `context-node` on the open element stack, thanks for spotting that 👍